shpc/main/modules/lmod.py

```diff
@@ -12,3 +12,13 @@ def __init__(self, **kwargs):
         """
         super(Client, self).__init__(**kwargs)
         self.module_extension = "lua"
+
+    def _module_sys_default_version(self, version_file, tag):
```

Comment:
We can set default as None here since it's not required.
```suggestion
    def _module_sys_default_version(self, version_file, tag=None):
```